bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -203,6 +203,11 @@ spec:
         kind: AtlasBackupSchedule
         name: atlasbackupschedules.atlas.mongodb.com
         version: v1
+      - description: AtlasCustomRole is the Schema for the AtlasCustomRole API
```

Comment:
nit
We should probably be more consistent with the docs here, this description is quite sparse while the endpoint one is very complete.